soroban-sdk/src/tests/budget.rs

```diff
@@ -0,0 +1,29 @@
+use crate as soroban_sdk;
+use soroban_sdk::{contractimpl, testutils::budget::CostType, Bytes, Env};
+
+pub struct Contract;
+
+#[contractimpl]
+impl Contract {
+    pub fn add(e: Env) -> Bytes {
+        let mut b = Bytes::from_array(&e, b"abcdefghijklmnopqrstuvwyxz");
```

Comment:
What is this bytes object?

Reply:
looks like it is a filler for the test

Reply:
> What is this bytes object?

It is the SDK Bytes type.

Reply:
Oh I see, this is just testing `BytesAppend`. Makes sense.